src/NuGet.Core/NuGet.CommandLine.XPlat/DeleteCommand.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.IO;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Dnx.Runtime.Common.CommandLine;
+using NuGet.Commands;
+using NuGet.Configuration;
+using NuGet.Logging;
+using NuGet.Protocol.Core.Types;
+using NuGet.Protocol.Core.v3;
+using System.Threading;
+
+namespace NuGet.CommandLine.XPlat
+{
+    class DeleteCommand : Command
+    {
+        public DeleteCommand(CommandLineApplication app, Func<ILogger> getLogger)
+        {
+            app.Command("delete", delete =>
+            {
+                delete.Description = Strings.Delete_Description;
+
+                var source = delete.Option(
+                    "-s|--source <source>",
+                    Strings.Source_Description,
+                    CommandOptionType.SingleValue);
+
+                //TODO: communicate to Nuget team, this should go to a top level argument, and
+                //should not be handled by individual cmdlet
```

Comment:
Top level arguments need to be written out before the name. Ex: xplat.exe --non-interactive delete packageId --source mySource

It's a confusing format, so I don't think this should ever be done.